src/libraries/System.Speech/src/Internal/Synthesis/TextFragmentEngine.cs

```diff
@@ -195,10 +195,10 @@ public void ProcessUnknownElement(object voice, ref FragmentState fragmentState,
         public void StartProcessUnknownAttributes(object voice, ref FragmentState fragmentState, string element, List<SsmlXmlAttribute> extraAttributes)
         {
             StringBuilder sb = new();
-            sb.AppendFormat(CultureInfo.InvariantCulture, "<{0}", element);
```

Comment:
Please revert all of the changes to System.Speech.  We've been avoiding churning the code, as it's here only for compat and doesn't have a robust set of tests.

Reply:
Done.